template-compose/app/src/androidTest/java/co/nimblehq/template/compose/ui/screens/home/HomeScreenTest.kt

```diff
@@ -0,0 +1,35 @@
+package co.nimblehq.template.compose.ui.screens.home
+
+import androidx.activity.compose.setContent
+import androidx.compose.ui.test.*
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import co.nimblehq.template.compose.R
+import co.nimblehq.template.compose.ui.AppDestination
+import co.nimblehq.template.compose.ui.screens.MainActivity
+import org.junit.*
+import org.junit.runner.RunWith
+
+class HomeScreenTest {
+
+    @get:Rule
+    val composeRule = createAndroidComposeRule<MainActivity>()
+
+    private var expectedAppDestination: AppDestination? = null
+
+    @Before
+    fun setup() {
```

Comment:
(Optional) For consistency with ViewModel tests
```suggestion
    fun setUp() {
```

Reply:
Fixed in https://github.com/nimblehq/android-templates/pull/384/commits/c80193ac0804bab5a875096becc6b3e5d0adb46d